docker/policies/port22.rego

```diff
@@ -0,0 +1,34 @@
+package appshield.dockerfile.DS004
+
+__rego_metadata__ := {
+    "id": "DS004",
+    "title": "Exposing Port 22",
+    "version": "v1.0.0",
+    "severity": "Medium",
+    "type": "Dockerfile Security Check",
+    "description": "Exposing Port 22 allows users to SSH inside the container.",
+    "recommended_actions": "Remove port 22 from the dockerfile",
+}
+
+denyList =[22] 
+
+# failPortCheck is true if the image contains an expose statement for value 22
+# tag is latest.
```

Comment:
cruft?

Reply:
What do you mean by "tag is latest"? Seems like a copy & paste issue.